lib/src/card/Card.tsx

```diff
@@ -14,48 +14,46 @@ const DxcCard = ({
   onClick,
   imageCover = false,
   margin,
-  contentPadding,
   tabIndex = 0,
   outlined = true,
   children,
 }: CardPropsType): JSX.Element => {
   const colorsTheme = useTheme();
-
   const [isHovered, changeIsHovered] = useState(false);
 
-  const imageComponent = (
-    <ImageContainer imageBgColor={imageBgColor}>
-      <TagImage imagePadding={imagePadding} imageCover={imageCover} src={imageSrc}></TagImage>
-    </ImageContainer>
-  );
-
   return (
-    <StyledDxcCard
+    <Card
+      hasAction={onClick || linkHref ? true : false}
```

Comment:
We can have this condition in a variable to avoid repeating it.

Reply:
Remember that the variables defined inside a React component are always recalculated every time the component re-renders. Since the code repetition is not in enough places, I think it is not worth it (it would require a `useMemo`).